test/Amazon.Extensions.CognitoAuthentication.IntegrationTests/MfaAuthenticationTests.cs

```diff
@@ -170,7 +171,7 @@ public MfaAuthenticationTests()
                 }
                 catch(Exception ex)
```

Comment:
exception filters are great for such cases

```suggestion
                catch(Exception ex) when (ex.InnerException != null && (bufferExMsg.Equals(ex.InnerException.Message) || bufferExMsgSNSPublish.Equals(ex.InnerException.Message)))

```

Reply:
Done